SDPi_Supplement/asciidoc-converter/src/main/kotlin/org/sdpi/asciidoc/Util.kt

```diff
@@ -1,21 +1,20 @@
 package org.sdpi.asciidoc
 
 import org.apache.logging.log4j.kotlin.loggerOf
-import org.asciidoctor.ast.Section
 import org.asciidoctor.ast.StructuralNode
+import org.sdpi.asciidoc.model.RequirementLevel

```

Comment:
What was the need to include the .RequirementLevel support in the asciidoc-converter tool?

Reply:
This is supposed to be meta information, which can also be used to verify the existence of requirement keywords in the actual requirement texts.